Add LuaExtended

viluon opened the pull request:
[LuaExtended](https://github.com/viluon/LuaExtended) is a better Lua syntax definition and snippet set for Sublime Text 3.

FichteFoll commented:
`.sublime-syntax` support has been added with build 3092. Please adjust your version selector accordingly.

Regarding linter support, you should check out the [`syntax_map` setting](http://www.sublimelinter.com/en/latest/global_settings.html?highlight=syntax_map#syntax-map), which makes this a lot easier.
You can also voice your support of being able to provide this information in the package that provides the additional syntax here: https://github.com/SublimeLinter/SublimeLinter3/issues/431

viluon commented:
Thanks for the heads-up @FichteFoll, I have fixed the version requirement and updated the [README](https://github.com/viluon/LuaExtended/blob/master/README.md) with a new, simpler guide on linters.

FichteFoll commented:
[Semantic versions](http://semver.org) consist of exactly three numeric parts.

viluon commented:
... Yet GitHub itself suggest the 'v' prefix
> It’s common practice to prefix your version names with the letter `v`.

![](https://i.imgur.com/uedmbUm.png)
(Please ignore the custom theme)

[Semantic Versioning](http://semver.org/) uses it for tagging *its own* versions
![](https://i.imgur.com/ez0WTuw.png)

and Package Control supports 'v' prefixes just fine (until https://github.com/wbond/package_control/issues/887 is closed as accepted, if that happens).

It is therefore a question if packages *should* change their `tags` fields or not, and given that there is no clash with Package Control itself, it seems just fine to use `v`'s for version prefixes, at least from my standpoint @FichteFoll.

FichteFoll commented:
>To my understanding, using v in GitHub release tags is completely okay.

It is. The Package Control crawler detects versions prefixed with `v` just fine, just like with not prefix when `tags` is set to `true`. You can choose a different prefix for your tags if you with to use another one, for example `"tags": "version-"`, and then tag all your releases as `version-1.2.3`.

The reason why I complained initially was because you only had major and minor parts but not patch.

viluon commented:
@FichteFoll Oh, I am sorry! Not sure how I missed that :confused:. Well, thanks or merging!
